.github/workflows/high-impact-alert.js

```diff
@@ -0,0 +1,28 @@
+const {
+  slackNotification,
+  getLocalConfigs,
+} = require('./helpers.js');
+
+const main = async (params) => {
+  const { context } = params;
+
+  try {
+    if (context.payload.label.name === 'high-impact') {
+      const { html_url, number, title } = context.payload.pull_request;
+      console.log('High impact label detected, sending Slack notification');
+      slackNotification(`:alert: High Impact PR has been opened: <${html_url}|#${number}: ${title}>.` +
+         ` Please prioritize testing the proposed changes.`, process.env.SLACK_HIGH_IMPACT_PR_WEBHOOK);
+    } else {
+      console.log('No high impact label detected');
+    }
```

Comment:
I prefer habitually using early returns, as if/else cases don't scale well once you add one-two more nested if/else statements.

The spacing is off, but I'd go with something such as this:
```suggestion
    if (context.payload.label.name !== 'high-impact') {
      console.log('No high impact label detected');
      return
    }
    const { html_url, number, title } = context.payload.pull_request;
    console.log('High impact label detected, sending Slack notification');
    slackNotification(`:alert: High Impact PR has been opened: <${html_url}|#${number}: ${title}>.` + 
    ` Please prioritize testing the proposed changes.`, process.env.SLACK_HIGH_IMPACT_PR_WEBHOOK);
```